### DISCUSSION TITLE
不知道为什么安装不了

### DISCUSSION THREAD

**Discussion by everclover** [ywmoyue/biliuwp-lite#839] [General]
### Bug 描述

![image](https://github.com/user-attachments/assets/08d960d9-dd25-416b-81c5-201ac9de8120)
请问怎么解决这个问题呢，已经挺久了？

### 操作步骤

```plain text
1.安装应用
2.安装失败——应用安装程序无法安装包相关项
```

### 期望结果

能够正常安装

### 截图

![image](https://github.com/user-attachments/assets/1d903f45-dae9-4836-803d-d5c7dab44ebe)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

Windows 11 21H2 (Build 22000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
请查看安装教程

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/blob/master/document/install-readme.md

**Reply by Brausedog** [reply]
我与他有同样的问题，并且我确定是完全按照教程里操作的。安装完证书之后右键查看.appx文件属性的证书仍显示不可用，大概几个月前就出现这种情况了，所以我还在用之前的版本，我的两台电脑都有这个问题QAQ

**Reply by Brausedog** [reply]
抱歉，已经解决了，当我没问

**Reply by ywmoyue** [reply]
@Everclover 是否已解决？
